modules/core/02-client/keeper/keeper.go

```diff
@@ -413,3 +413,23 @@ func (k Keeper) GetClientStatus(ctx sdk.Context, clientState exported.ClientStat
 	}
 	return clientState.Status(ctx, k.ClientStore(ctx, clientID), k.cdc)
 }
+
+// GetParams returns the total set of ibc-client parameters.
+func (k Keeper) GetParams(ctx sdk.Context) types.Params {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get([]byte(types.ParamsKey))
+	if len(bz) == 0 {
+		return types.Params{}
+	}
```

Comment:
we should panic on unset params? Similar to how the other param prs do

Reply:
there's been quite some back and forth here w/o much consensus. See some discussions on connections params https://github.com/cosmos/ibc-go/pull/3650#discussion_r1210398066 and here in this pr https://github.com/cosmos/ibc-go/pull/3640#discussion_r1204210620.

Reply:
done https://github.com/cosmos/ibc-go/pull/3721